vulkan_helpers/helper_functions.h

```diff
@@ -299,6 +299,11 @@ uint32_t inline GetMemoryIndex(VkDevice* device, logging::Logger* log,
     }
     break;
   }
+  // If we couldn't find a device-local memory type, fall back to a non-device
```

Comment:
should there be any sort of warning or indication that this occurred? I'm not sure how big of a deal this would be in practice.

Reply:
For the test applications I don't this this is a terribly large issue.